app/kumactl/cmd/root.go

```diff
@@ -87,7 +87,7 @@ func NewRootCmd(root *kumactl_cmd.RootContext) *cobra.Command {
 			if kumaBuildVersion == nil {
 				cmd.PrintErr("WARNING: Unable to confirm the server supports this kumactl version\n")
 			} else if kumaBuildVersion.Version != kuma_version.Build.Version || kumaBuildVersion.Tagline != kuma_version.Product {
-				cmd.PrintErr("WARNING: You are using kumactl version " + kuma_version.Build.Version + " for " + kuma_version.Product + ", but the server returned version: " + kumaBuildVersion.Tagline + " " + kumaBuildVersion.Version + "\n")
+				cmd.PrintErr("WARNING: You are using kumactl version " + kuma_version.Build.Version + " for " + kuma_version.Product + ", but the server returned version: " + kumaBuildVersion.Tagline + " " + kumaBuildVersion.Version + " Server Instance Id: " + kumaBuildVersion.InstanceId + " Service Cluster Id: " + kumaBuildVersion.ClusterId + "\n")
```

Comment:
I don't think it make sense to print this instance ID and cluster ID in a warning. Was that requested by someone?

Reply:
This was requested, but may no longer be necessary now that we have a warning when we cannot retrieve the version. I think the original idea was that we may have been "talking to the wrong cluster" or something, which was a misdiagnosis of confusion about the warning.

Reply:
Update - no longer necessary. Will remove.